Configure

```diff
@@ -1671,6 +1671,13 @@ while (<PIPE>) {
 }
 close(PIPE);
 
+# Xcode did not handle $cc -M before clang support
+my $makedepprog;
+if ($predefined{__GNUC__} >= 3 && !(defined($predefined{__APPLE_CC__})
+                                    && !defined($predefined{__clang__}))) {
```

Comment:
Well, if you play `defined($predefined{})` game, then it makes more sense to check for `defined($predefined{__GNUCC__})`. Because undefined $foo in *logical* expression implies check for defined. I mean `use warnings; my $a; 1 if ($a)` triggers no warnings. Of course in `__GNUC__` case it's sufficient to `($predefined{__GNUC__} // -1)` instead of dragging whole defined().

Reply:
When checking the value of `$predefined{__GNUC__}`, the rest of the Configure script did not include `defined()`. I'm just following existing usage here. :)

Reply:
Do note that I said "*if* you play the game" :-)